cellprofiler_core/setting/_image_plane.py

```diff
@@ -55,7 +55,10 @@ def build(url, series=None, index=None, channel=None):
         )
 
     def __get_field(self, index):
-        f = self.value_text.split(" ")[index]
+        split = self.value_text.split(" ")
+        if index >= len(split):
```

Comment:
shouldn't this be > rather than >= ? something that has a len of 1 will have no index 1

Reply:
With len of 1 only index 0 is valid, so index >=1 is invalid. For len 2 indexes 0 and 1 are valid, so if index == len (2) that's still invalid.

Reply:
Oh, I'm having a brain fart, I was reading it as that not being the path to None, rather than that being the path away from none. My bad.